database/insertData/03_template_A_featureShowcase.js

```diff
@@ -689,12 +688,55 @@ exports.queries = [
                         layout: "inline"
                         checkboxes: {
                           operator: "API"
+                          children: [
+                            "https://jsonplaceholder.typicode.com/users"
+                            []
+                            "name"
+                          ]
+                        }
+                      }
+                    }
+                    {
+                      index: 30
+                      code: "Q2GraphQL"
+                      title: "Country code"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: {
+                        label: "Enter 2 letters code of a valid country"
+                      }
+                    }
+                    {
+                      index: 31
+                      code: "TXT_COUNTRY"
+                      title: "Country name"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: {
+                        text: {
+                          operator: "graphQL",
+                          children: [
+                            "query country ($code: ID!) { country (code: $code) { name } }",
+                            "https://countries.trevorblades.com",
+                            [ "code" ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "responses.Q2GraphQL.text"
+                                ""
+                              ]
+                            }
+                            "country"
+                          ]
+                        }
+                        visibilityCondition: {
+                          operator: "!="
```

Comment:
As we discussed, this won't work with the currently installed package, so maybe save this addition to another PR once the package has been updated.

Reply:
Done: 30a1c6c